assets/styles/card.css

```diff
@@ -0,0 +1,22 @@
+@import url(colors.css);
+
+/* CARD STYLES */
+.card {
+    width: 310px;
+    height: 342px;
+
+    background-color: var(--white);
+    border-radius: 20px;
+}
+
+.card__content {
+    padding: 20px;
+
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
+
+.card__content > .header-3 {
```

Comment:
"header-3" - what exactly does this name mean? is it about all h3 or about order in HTML layout?